src/library_pipefs.js

```diff
@@ -5,9 +5,7 @@
  */
 
 mergeInto(LibraryManager.library, {
-  $PIPEFS__postset: function() {
-    addAtInit('PIPEFS.root = FS.mount(PIPEFS, {}, null);');
-  },
+  $PIPEFS__postset: () => addAtInit('PIPEFS.root = FS.mount(PIPEFS, {}, null);'),
```

Comment:
I guess for consistency it's ok to do this for postsets as well, but they are evaluated at compile time so this does not affect the generated code.